website/docs/r/machine_learning_workspace.html.markdown

```diff
@@ -185,21 +185,19 @@ The following arguments are supported:
 
 * `description` - (Optional) The description of this Machine Learning Workspace.
 
-* `discovery_url` - (Optional) The URL for the discovery service to identify regional endpoints for machine learning experimentation services.
```

Comment:
how come we are removing this? it is still in schema?

Reply:
It's a computed property and it's already in `Attributes Reference` section.